graphics/nine_patch.cpp

```diff
@@ -186,13 +184,43 @@ NinePatchBitmap::NinePatchBitmap(Graphics::TransparentSurface *bmp, bool owns_bi
 	_titleWidth = 0;
 	_titlePos = titlePos;
 
+	const uint32 key = bmp->getTransparentColor();
+	uint32 black, white;
+
+	if (bmp->format.isCLUT8()) {
+		uint8 palette[256 * 3];
+		bmp->grabPalette(palette, 0, 256);
+
+		black = 0;
+		white = 1;
```

Comment:
Should these two be set to uint32(-1) for following bad_bitmap checks?

Reply:
Good catch. Fixed.